docs/pages/Builder.md

```diff
@@ -21,6 +21,7 @@ All available standard tasks are documented [in the API reference](https://sap.g
 
 | Task                           | Type `application` | Type `library` | Type `theme-library` |
 | ------------------------------ | :----------------: | :------------: | :------------------: |
+| enhanceManifest                | *enabled*          | *enabled*      |                      |
```

Comment:
Is this the actual order of execution?

Reply:
Nope. I thought its sorted alphabetical but its not. So, I have moved the newly added row to the correct place.